index.html

```diff
@@ -9,11 +9,66 @@
     <link rel="stylesheet" href="css/style.css">
     <link rel="shortcut icon" type="image/x-icon"  href="favicon.ico">
 </head>
+
 <body>
+  <header>
+    <img class="img" src="./img/karma-logo.svg" alt="Logo"></img>
+    <nav class="logo1">
+      <ul class="list">
+        <li><a class="bar" href="index.html">Meet Karma</a></li>
+        <li><a href="#">How it Works</a></li>
+        <li><a href="store.html">Store</a></li>
+        <li><a href="#">Blog</a></li>
+        <li><a href="#">Help</a></li>
+        <li><a href="#">Login</a></li>
+      </ul>
+    </nav>
+  </header>
+  
+    <main>
+      <div class="karma">
+        <h1> Introducing karma <h1>
+        <p> Bring WiFi with you, everywhere you go.<p>
+        <button><a href="#">Learn More</a></button>  
+      </div>
+     
+    </main>
+          <p>Everyone needs a little Karma</p>
+
+    <article>
+        <div>
+          <img src="./img/icon-devices.svg" alt="devices">
+          <h5>Internet for all devices</h5>
+        </div>
+        
+        <div>
+          <img src="./img/icon-coffee.svg" alt="coffee">
+          <h5>Boots your productivity</h5>
+        </div>
+
+       <div>
+          <img src="./img/icon-refill.svg" alt="refill">
+          <h5>Pas as You Go</h5>
+       </div>
+
+      </div>
+
```

Comment:
Not a huge issue again, but let's try to ensure that there is consistency with the whitespace and indentation in your code - automated tools are great here, but it's also good to know the rules for yourself as well.